tests/positive/files/directory.go

```diff
@@ -21,6 +21,7 @@ import (
 
 func init() {
 	register.Register(register.PositiveTest, CreateDirectoryOnRoot())
+	register.Register(register.PositiveTest, ForceDirCreation())
```

Comment:
Is it worth adding a test which does a force dir creation over a pre-existing non-empty directory?

Reply:
More tests are always worth it. Added.